declarations.d.ts

```diff
@@ -2,3 +2,15 @@ declare module "*.mp4" {
   const src: string;
   export default src;
 }
+declare module "*.gif" {
+  const src: string | StaticImageData;
+  export default src;
+}
+declare module "*.png" {
+  const src: string |StaticImageData
+  export default src;
+}
+declare module "*.svg" {
+  const src: string;
+  export default src;
+}
```

Comment:
기여해주셔서 감사합니다 :-)
현재 빠르게 개발을 하느라 곧 추가될 기능만 마무리되면 SVG 컴포넌트들을 각 개별 컴포넌트로 분리할 예정인데요.
작성해주신 `string` 타입에 추가로 컴포넌트 타입도 같이 추가하면 좋을 것 같습니다!

```suggestion
declare module "*.svg" {
  export const ReactComponent: React.FunctionComponent<
    React.SVGProps<SVGSVGElement>
  >;
  const src: string;
  export default src;
}
```

Reply:
@klmhyeonwoo I’ve updated the commit as you suggested~

Reply:
Good !